File: mario-python/mario.py
from cs50 import get_int

while (1):
    height = get_int("height: ")
    if (height > 0 and height < 9):
        break
for i in range(1, height+1):
    print(" " * (height - i), end="")
    for j in range(2):
        print("#" * i, end="")
        if j == 0:
            print("  ", end="")
    print()

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void printBlocks(int);
int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);
    for (int i = 1; i < height + 1; i++)
    {
        for (int j = height - i; j > 0; j--)
        {
            printf(" ");
        }
        printBlocks(i);
        printf("\n");
    }
}
// Prints "lenght" hastags two times with two gaps between and no gaps at the end
void printBlocks(int lenght)
{
    for (int j = 0; j < 2; j++)
    {
        for (int i = 0; i < lenght; i++)
        {
            printf("#");
        }
        if (j == 0)
        {
            printf("  ");
        }
    }
}
